lms/services/grading_info.py

```diff
@@ -25,33 +26,48 @@ def __init__(self, _context, request):
 
     def get_by_assignment(self, oauth_consumer_key, context_id, resource_link_id):
         """
-        Get available grading data for students for an assignment.
+        Return all the (GradingInfo, HUser) tuples for a given assignment.
 
-        Retrieve all :class:`~lms.models.GradingInfo`s that match this
-        assignment (each unique combination of (``oauth_consumer_key``,
-        ``context_id``, ``resource_link_id``) corresponds to an assignment).
-        There should be one record per applicable student who has launched this
-        assignment (and had ``GradingInfo`` data persisted for them).
+        Return a list of all the (GradingInfo, HUser) tuples for the assignment
+        identitied by the given params.
```

Comment:
This seems a bit redundant with the above. Maybe just remove this?

Reply:
Removed